### PR TITLE
ref(style): Updated frame list style

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Type:** Style Refactor

**Description:** 

- Updated frame list styles to look better on regular iPad
- Changed the logic of displaying the right issues sidebar (the one with charts)
- Fixed other component styles to make them look better on all devices

### CODE REVIEW

**Comment by matejminar** [comment]
I see there is the right sidebar moved to the bottom already at 1280px - isn't that kind of soon?

1280px seems to me like a pretty common resolution, we should provide good desktop experience there.

**Comment by priscilawebdev** [comment]
@matejminar you are right. I have decreased it to 1200px 😉thanks for the feedback

**Comment by matejminar** [comment]
Few stacktraces still don't look great - mainly ones with addresses.
In percy those are f.e: `issue details rust`, `issue details cocoa`.

Alignment seems off
image

Could you please check that out?

**Comment by markstory** [comment]
Python stack traces are showing a dropped bottom padding on the tags section.

Screen Shot 2020-03-23 at 11 57 19 AM

**Comment by priscilawebdev** [comment]
@markstory yes I have seen it and already fixed 😉

**Comment by jan-auer** [comment]
There are a few changes that look rather unfortunate, for instance _"issue details cocoa"_:

image

Or _"issue details rust"_:

image

**Comment by priscilawebdev** [comment]
So..everything looks fine now  😉

image

**Review by billyvg — COMMENTED** [review]
Codewise this looks good, but can we get @getsentry/design sign-off on these new breakpoints?

**Comment by billyvg** [comment]
I have not pulled this locally to check how it looks at the different breakpoints, we should note that percy doesn't have snapshots for all breakpoints

File: src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx [hunk 1]
```diff
@@ -211,7 +212,7 @@ export class Frame extends React.Component {
 
   renderLeadHint() {
     if (this.leadsToApp() && !this.state.isExpanded) {
-      return <span className="leads-to-app-hint">{'Called from: '}</span>;
+      return <LeadHint className="leads-to-app-hint">{'Called from: '}</LeadHint>;
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx]: If this is now a styled component can we remove the classname?

File: src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx [hunk 2]
```diff
@@ -381,24 +384,36 @@ const RepeatedContent = styled(VertCenterWrapper)`
...
-const NativeLineContent = styled(VertCenterWrapper)`
-  flex: 1;
-  overflow: hidden;
-  justify-content: center;
-
-  & > span {
-    display: block;
-    padding: 0 5px;
+const PackageInfo = styled('div')`
+  display: flex;
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx]: Does this component have two display definitions?

File: src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx [hunk 3]
```diff
@@ -381,24 +384,36 @@ const RepeatedContent = styled(VertCenterWrapper)`
...
+  order: 2;
+  @media (min-width: ${props => props.theme.breakpoints[2]}) {
+    order: 0;
   }
+  display: grid;
+  grid-template-columns: auto 1fr;
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx]: 🤔 is it flex or display?

File: src/sentry/static/sentry/app/components/tooltip.tsx [hunk 4]
```diff
@@ -233,7 +225,17 @@ class Tooltip extends React.Component<Props, State> {
...
+// than the elements it is wrapping
+const Container = styled('span')<{
+  containerDisplayMode?: React.CSSProperties['display'];
+}>`
+  ${p => p.containerDisplayMode && `display: ${p.containerDisplayMode}`};
+  max-width: 100%;
+`;
+
+type TooltipContentProps = Pick<Props, 'popperStyle'>;
+const TooltipContent = styled('div')<TooltipContentProps>`
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/tooltip.tsx]: Why the change here, having this inline seemed fine.

File: src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx]: it's grid... flex was there by mistake :)

File: src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx]: I need to be very careful in this part of the code and this is something I prefer to change later

File: src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx (hunk 3 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx]: do you mean grid? display:flex was there by mistake... I just removed it 😉

File: src/sentry/static/sentry/app/components/tooltip.tsx (hunk 4 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/tooltip.tsx]: yes, it is fine ... I have to investigate because with the current option, this part of the code in my vscode looks strange:
image

**Comment by priscilawebdev** [comment]
Thank you all for the feedback! :) @billyvg I checked the changes on other breakpoints and looks great... but yes, it would be nice if you could check it too

**Comment by jan-auer** [comment]
Percy screenshot from the mobile viewport:
image

- Can we left-align the address here? The indentation looks a bit odd and uses up space that could be used for rendering the image name.
- Can you use ellipsis and avoid wrapping of the image name? Like this:

```
| +0x0085ac    sentry-ios-cocoap... |
| ViewController.causeCrash         |
```

Also, what would you think if we collapse the frames by default on mobile viewports?

**Comment by matejminar** [comment]
@jan-auer @priscilawebdev I think addresses are not completely left aligned because of the relative/absolute switcher that shows up only on hover - which is irrelevant on the mobile experience. Do we want to support converting of addresses on mobile?

**Comment by priscilawebdev** [comment]
@jan-auer 

> Can we left-align the address here? The indentation looks a bit odd and uses up space that could be used for rendering the image name.
 The empty space is there because of a switcher: 
image

Should I hide it on small devices?

> Can you use ellipsis and avoid wrapping of the image name? 
Yes I can do that

**Comment by matejminar** [comment]
I've noticed that we put the right sidebar all the way down on < 1200px, but all the way up on < 768 - is changing the order like that expected behavior?

Also on < 1200 we collapse stacktraces to mobile experience which is too soon imho - considering there is much more space than before (because the right sidebar is not there anymore).

**Comment by priscilawebdev** [comment]
@matejminar thank you for your suggestions.. I totally agree with them and I've just pushed another commit with some changes 😉

@jan-auer  I have also applied your suggestions... Hope it's better now ...thanks 🙂

**Comment by matejminar** [comment]
Cool, I think we are getting there :) 

We are hiding toggle thingy for relative/absolute addresses at < 1200px, however, addresses don't go all the way to left until <768. Can we please hide it then?

**Review by matejminar — APPROVED** [review]
Great work 🎉
Tested locally and it looks good to me.
